dot/core/service.go

```diff
@@ -491,6 +491,10 @@ func (s *Service) ProcessBlockAnnounceMessage(msg network.Message) error {
 		return err
 	}
 
+	// send block request message
+	log.Debug("[core] sending new block to syncer", "number", blockAnnounceMessage.Number)
+	s.blockNumOut <- blockAnnounceMessage.Number
+
```

Comment:
remove this, this is duplicate code of line 488

Reply:
done